### ISSUE TITLE
There should be a way to reset the Activity Designer back to "defaults"

### ISSUE THREAD

**Issue by cbradyatinquire** [MathNetDev/MathNetClient#33]
(this happens

**Comment by ghost** [comment]
Added a reset button for this.

**Comment by mtstevens** [comment]
Tested in Test, 10/4/18 @ 10:30am PDT. Test conditionally successful.

The successful part...
The "Reset View" button effectively deletes all elements in the GeoGebra pane(s), resets the visible panes back to Algebra and Graphics1 (see "The conditions..." below), and resets the zoom level of the axes in Graphics1. Several elements were tested:
- In Graphics1: points, lines (including 'quick construction' lines using tools like "Parallel Line" and "Line of Best Fit"), polygons, conics, sliders, text boxes, check boxes... they all were deleted.
- In Algebra: in addition to things tied to elements in Graphics View (e.g., the ordered pairs of the points), even functions that were directly typed in (e.g., y=2x+1) were deleted.
- In Graphics2, Spreadsheet, and 3D Graphics: elements were deleted (evidenced by opening those panes after resetting and seeing blank panes).

The conditions...
1) If either Graphics2 or 3D Graphics are visible, and "Reset View" is clicked, everything resets as described above; however, there is a "dead space" on the right side of Graphics1 where nothing can happen (not even menus open up on this space). See screenshots below.
<img width="1125" alt="screen shot 2018-10-04 at 10 34 08 am" src="https://user-images.githubusercontent.com/41483974/46492580-d0796680-c7c2-11e8-930e-82ddae8a05a6.png">
<img width="1093" alt="screen shot 2018-10-04 at 10 46 35 am" src="https://user-images.githubusercontent.com/41483974/46492590-d7a07480-c7c2-11e8-82b4-368dccb84a2b.png">
2) If any saved constructions are used (via the "Use Construction" button), and then "Reset View" is clicked, points that are subsequently plotted in Graphics1 do not have the "unassigned" caption.

**Comment by ghost** [comment]
Thanks @mtstevens for your feedback ! I think both the issues you've mentioned should be fixed now on test. Please check and let me know when possible. Thanks.

**Comment by IsaacNPsmash** [comment]
Tested 10/11/18 @ 8:47 am Test Branch 
I tested both issues @mtstevens described and both seem to be fixed.
Test1 Reset view with multiple perspectives: I tested the reset view button with CAS, Graphics2, 3D Graphics, Spreadsheet, Probability calculator, and Construct Protocol, and all reset with no empty space. I also tested it after closing Graphics1 and Algebra, as well as with two and three added perspectives (totalling 4 and 5). All worked seamlessly without empty space.
Test2 Reset view after using construction: I tested the reset view after implementing the "Use Construction." Everything reset well and points added both before and after clicking the reset button had caption "unassigned."
Test3 Combined: I also used a construction, added 3D Graphics, and then hit the reset view button. All seemed to go well, all constructs were delete and the view went back to Algebra-Graphics1 with not empty space.
